src/BatchEventData.ts

```diff
@@ -0,0 +1,151 @@
+import Log from './helpers/Logger';
+import { isString, isNumber, isBoolean } from './helpers/TypeHelpers';
+
+const Consts = {
+  AttributeKeyRegexp: /^[a-zA-Z0-9_]{1,30}$/,
+  EventDataMaxTags: 10,
+  EventDataMaxValues: 10,
+  EventDataStringMaxLength: 64,
+};
+
+export enum TypedEventAttributeType {
+  String = 's',
+  Boolean = 'b',
+  Integer = 'i',
+  Float = 'f',
```

Comment:
Please use a full name (`float`) for the type key. When looking at the iOS and Android code it is not explicit enough that "f" references a "float" type.

Reply:
ok